Sources/performance-worker/WorkerService.swift

```diff
@@ -87,7 +90,29 @@ final class WorkerService: Grpc_Testing_WorkerService.ServiceProtocol, Sendable
   ) async throws
     -> GRPCCore.ServerResponse.Stream<Grpc_Testing_WorkerService.Method.RunServer.Output>
   {
-    throw RPCError(code: .unimplemented, message: "This RPC has not been implemented yet.")
+    return ServerResponse.Stream { writer in
+      for try await message in request.messages {
+        switch message.argtype {
+        case let .some(.setup(serverConfig)):
+          try await self.serverSetup(serverConfig)
+        case let .some(.mark(mark)):
+          let response = try await self.makeStatsResponse(for: mark)
+          try await writer.write(response)
+        case .none:
+          ()
+        }
+        throw RPCError(code: .unavailable, message: "This RPC has not been implemented yet.")
```

Comment:
Why are we still throwing an error here?

Reply:
Forgot to delete it, thanks!